File: programmers/42842/42842.py
def solution(brown, yellow):
    answer = []
    total = brown+yellow
    h = 3
    while True :
        w = total // h
        if (w-2)*(h-2) == yellow :
            answer = [w,h]
            break
        else :
            h += 1
        
    return answer

File: C++/programmers/42842/42842.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int brown, int yellow) {
    int w = 0;
    int total = brown + yellow;
    for(int h = 3 ; h*h <= total ; ++h) {
        if(total%h != 0) continue;
        w = total / h;
        if((h-2)*(w-2) == yellow)
            return {w,h};
    }
    return {};
}